lib/active_model/serializer/caching.rb

```diff
@@ -79,13 +77,6 @@ def non_cached_attributes
           _attributes - cached_attributes
         end
 
-        # @api private
-        # Used by FragmentCache on the CachedSerializer
-        #  to call attribute methods on the fragmented cached serializer.
-        def fragmented(serializer)
-          self._fragmented = serializer
-        end
```

Comment:
I found this method more confusing than helpful (And I'd like to remove _fragmented entirely!) https://github.com/rails-api/active_model_serializers/pull/1781/files#r66013775